mongoimport/options.go

```diff
@@ -44,6 +44,9 @@ type InputOptions struct {
 
 	// Indicates that field names include type descriptions
 	ColumnsHaveTypes bool `long:"columnsHaveTypes" description:"indicated that the field list (from --fields, --fieldsFile, or --headerline) specifies types; They must be in the form of '<colName>.<type>(<arg>)'. The type can be one of: auto, binary, bool, date, date_go, date_ms, date_oracle, double, int32, int64, string. For each of the date types, the argument is a datetime layout string. For the binary type, the argument can be one of: base32, base64, hex. All other types take an empty argument. Only valid for CSV and TSV imports. e.g. zipcode.string(), thumbnail.binary(base64)"`
+
+	// Indicates that the legacy extended JSON format should be used to parse JSON documents. Defaults to false.
+	Legacy bool `long:"legacy" default:"false" description:"use the legacy extended JSON format (defaults to 'false')"`
```

Comment:
True should be an error unless `--type=json`.  Please add code and tests for that case.

Reply:
Added in the new commit.